checkstyle.xml

```diff
@@ -53,7 +53,10 @@
         <module name="TypeName"/>
         <module name="IllegalImport"/> <!-- defaults to sun.* packages -->
         <module name="IllegalImport">
-            <property name="illegalPkgs" value="clover, junit.framework, org.apache.commons.lang, org.slf4j, javax.annotation.Nonnull, com.google.common.collect"/>
+            <property name="illegalPkgs" value="clover, junit.framework, org.apache.commons.lang, org.slf4j, com.google.common.collect"/>
+        </module>
+        <module name="IllegalImport">
+            <property name="illegalClasses" value="javax.annotation.Nonnull"/>
```

Comment:
It is possible to group this <property> element under the other <module name="IllegalImport"> entry (with clover, junit.framework, etc), I don't think we need three <module name="IllegalImport"> entries but we do still need the self closing module on line 54 as I just tested this locally and sun.* imports get accepted without it.

Reply:
probably? I copied how we handle this in our main repo, I don't mind either way to be honest.

Reply:
I prefer grouping them but it's a much of a muchness 🤷